clap_builder/src/builder/command.rs

```diff
@@ -1792,7 +1792,8 @@ impl Command {
     /// This is often used to describe how to use the arguments, caveats to be noted, or license
     /// and contact information.
     ///
-    /// If [`Command::after_help`] is not specified, this message will be displayed for `-h`.
+    /// This message is not displayed for `-h`. Use [`Command::after_help`] to show message
```

Comment:
Let's make this consistent with `long_about` (we don't need to mention `-h` because we explicitly said this is for `--long`)

Reply:
I'm afraid I'm not sure I understand what you want me to do. Are you suggesting that I remove the whole sentence here and in `long_about`?

Reply:
`long_about` doesn't say this.  Instead it only says

> If [Command::about](https://docs.rs/clap/latest/clap/struct.Command.html#method.about) is not specified, this message will be displayed for -h.

Let's make this more like `long_about`.